README.md

```diff
@@ -4,33 +4,64 @@ Software Engineering Day1 Assignment
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
-
+Software engineering is the application of engineering principles to design,develop,test and deploy softwate systems.
+It helps to create software application in various aspects of modern life,such as commerce,communication,entertainmentvand health care.
 
 Identify and describe at least three key milestones in the evolution of software engineering.
-
+Development of programming languages
+Establishment of software engineering as a discipline in the 1960's
```

Comment:
1. Planning: The initial stage of software development, Planning, involves defining the software’s purpose and scope, much like pinpointing our destination and plotting the best route. We uncover the tasks at hand during this phase and strategize for efficient execution.
2. Requirement Analysis: Requirements Analysis, seeks to identify and record the precise requirements of the final users. In this phase, the team is looking to answer, “What are the expectations of our users from our software?” This is called requirements gathering.
3. Designing: The Design phase is all about building the framework. The development team is responsible for software engineering and outlines the software’s functionality and aesthetic. This ultimately results in the software product. The emphasis lies on outlining the software’s structure, navigation, user interfaces, and database design. This phase ensures that the software is user-friendly and performs its tasks efficiently.
4. Coding: This phase is when engineers and developers get down to business and start converting the software design into tangible code.
5. Testing: Software testing involves a thorough examination of the software for any bugs or glitches that might have slipped through during coding. The aim is to ensure flawless software operation before it reaches the end-users. And even identify opportunities for enhancement.
6. Deployment:  The Deployment phase involves rolling out the meticulously tested and fine-tuned software to its end-users.
7. Maintenance: The maintenance phase is characterized by constant assistance and improvement, which guarantees the software’s best possible functioning and longevity and ensures it meets customer expectations.